src/styles/blocks/main.scss

```diff
@@ -0,0 +1,20 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  font-family: $main-font-family;
+}
+
+.card__informacion {
+  margin: 0 16px 16px;
+  width: 166px;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
+  top: 1px;
+  left: 1px;
```

Comment:
The `top` and `left` properties will have no effect unless the element has a positioning context. Consider adding `position: relative;` or another positioning property if needed.